lib/rewriter.js

```diff
@@ -3,10 +3,13 @@
 const assert = require('assert');
 const ts = require('typescript');
 const tsMorph = require('ts-morph');
-const cleanConsole = require('./mods/clean-console.js');
-const replaceConstructors = require('./mods/replace-constructors.js');
-const replaceMethods = require('./mods/replace-methods.js');
 const {addAllImports} = require('./imports.js');
+const {
+  cleanConsole,
+  replaceConstructors,
+  replaceMethods,
+  replaceGetSet,
+} = require('./mods');
```

Comment:
The index.js file seems like unnecessary redirection here. It just added code (the require statements still exist, over there).

I'd suggest just continuing with the current architecture. We could consider something where index.js contains an array of functions, and then this file loops over them and applies them all; that would potentially be a worthwhile abstraction. But let's do that as a separate PR, if we want to.